documentation/docs/13-cli.md

```diff
@@ -47,3 +11,7 @@ Like `svelte-kit dev`, it accepts the following options:
 For package authors, see [packaging](/docs/packaging). `svelte-kit package` accepts the following options:
 
 - `-w`/`--watch` — watch files in `src/lib` for changes and rebuild the package
+
+### svelte-kit sync
+
+`svelte-kit sync` creates the generated files for your project such as types and a `tsconfig.json`.
```

Comment:
we should note when this runs, and that people aren't expected to run it manually